lib/Data/Mapper/ContentUpdateMapper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Data\Mapper;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+use EzSystems\RepositoryForms\Data\Content\ContentUpdateData;
+use EzSystems\RepositoryForms\Data\Content\FieldData;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ContentUpdateMapper implements FormDataMapperInterface
+{
+    /**
+     * Maps a ValueObject from eZ content repository to a data usable as underlying form data (e.g. create/update struct).
+     *
+     * @param ValueObject|\eZ\Publish\API\Repository\Values\Content\Content $contentDraft
+     * @param array $params
+     *
+     * @return ContentUpdateData
+     */
+    public function mapToFormData(ValueObject $contentDraft, array $params = [])
```

Comment:
Side as it affects all mappers, and hence should be dealt with separated: *Given the generic nature of the type hint on these, they should probably check type inline before using the properties on the assumed objects.*